geometry_msgs/Pose target
float64[] force_threshold
---
bool success
string error_msg